### PR TITLE
Add PM2_ENABLED flag to Docker

### PR DESCRIPTION
**Pull request by jason-fox**
This PR is a potential fix for https://github.com/telefonicaid/iotagent-node-lib/issues/790

* Update entrypoint to use/not use pm2
* Add `PM2_ENABLED` to Docker README
* Update CNR.

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -150,6 +150,19 @@ COPY . /opt/iotasigfox/
...
+The IoT Agent within the Docker image can be run encapsulated within the [pm2](http://pm2.keymetrics.io/) Process
+Manager by adding the `PM2_ENABLED` environment variable.
+
+```console
+docker run --name iotagent -e PM2_ENABLED=true -d fiware/sigfox-iotagent
+```
+
+Use of pm2 is **disabled** by default. It is unnecessary and counterproductive to add an additional process manager if
+your dockerized environment is already configured to restart Node.js processes whenever they exit (e.g. when using
+Kubenetes)
```
Comment (fgalan) [inline docker/README.md]: Typo: Kubernetes

**Comment by fgalan** [comment]
The documentation sub-job in travis is failing. It would be great if you could take advantage of this PR to solve it as they seem to be trivial fixes. If you prefer not doing so it's also ok, just tell me and I'll do in a separate PR :)

File: docker/README.md (hunk 1 above)
Comment (jason-fox) [reply docker/README.md]: Fixed ae42bbdb

**Comment by jason-fox** [comment]
> The documentation sub-job in travis is failing. 

It looks like the Readthedocs DNS is configured incorrectly. The only thing which can be done here is to disable link checking for `*.rtfd.io` - fixed 6f0572b

**Comment by fgalan** [comment]
> The documentation sub-job in travis is failing.

What about changing `*.rtfd.io` -> `*.readthedocs.io` in links, as done in other cases, for instance https://github.com/telefonicaid/iotagent-json/commit/2b9fa26e3b37d7404c439a916f39b419891e4ada)? Better to use the same solution for all IOTA respositories...

**Pull request merged**